mapreduce/datadog_checks/mapreduce/data/conf.yaml.example

```diff
@@ -107,6 +107,11 @@ instances:
     #
     cluster_name: <MAPREDUCE_CLUSTER_NAME>
 
+    ## @param disable_legacy_cluster_tag - boolean - optional - default: false
```

Comment:
Can you make it:
- default: false
- example: true
- enabled: true

That way it will be enabled for new customers and disabled for existing ones